README.md

```diff
@@ -34,12 +34,22 @@ The operator will then create a CatalogSourceConfig CR which will, for the time
 
 #### In an OKD Cluster
 ```bash
-$ oc apply -f deploy/
+$ oc apply -f deploy/catalogsourceconfig.crd.yaml
+$ oc apply -f deploy/operatorsource.crd.yaml
+$ oc apply -f deploy/service_account.yaml
+$ oc apply -f deploy/role.yaml
+$ oc apply -f deploy/role_binding.yaml
+$ oc apply -f deploy/operator.yaml
 ```
 
 #### In a Kubernetes Cluster
 ```bash
-$ kubectl apply -f deploy/
+$ kubectl apply -f deploy/catalogsourceconfig.crd.yaml
+$ kubectl apply -f deploy/operatorsource.crd.yaml
+$ kubectl apply -f deploy/service_account.yaml
+$ kubectl apply -f deploy/role.yaml
+$ kubectl apply -f deploy/role_binding.yaml
+$ kubectl apply -f deploy/operator.yaml
 ```
 
 ### Deploying the Marketplace Operator with OLM
```

Comment:
Is deploying Marketplace in a cluster without OLM a planned use case? Is it intended that Marketplace could potentially function without OLM?

Reply:
Marketplace cannot function without OLM. However, it can be deployed in multiple ways which is what we are calling out here. We will soon add docs for using a subscription object to deploy once we are part of the OLM rh-operators CatalogSource.